WPFSKillTree/SkillTreeFiles/AscendancyClasses.cs

```diff
@@ -3,14 +3,9 @@
 
 namespace POESKillTree.SkillTreeFiles
 {
-    public class AscendancyClasses
+    public static class AscendancyClasses
```

Comment:
The hardest one...
Ideally I also would like to avoid new static classes introduction. To reduce problems while usage and testing I made them re-initializable. `BuildConverter` (assuming you advice for default deserializer) doesn't hide its dependencies: every delegate it calls it receives from the outside. `AscendancyClasses` looks worse.
I'll see what I can do but I assume it will be just different at the moment, not better. The SkillTree is responsible for initialization of lots of things and interfaces extraction doesn't realy help, because you still need to access the SkillTree to get needed reference while these static classes available independently.